page-index/page-index.js

```diff
@@ -61,9 +61,9 @@ const pageIndex = [
                         href: "migrate-from-postgresql",
                         to: "/getting-started/migrating-data"
                     }, {
-                  			Title: "Data Node Authentication",
-                  			type: PAGE,
-                        href: "data-node-authentication",
+                  	Title: "Data Node Authentication",
+                  	type: PAGE,
+                        href: "/getting-started/setup/data-node-authentication",
```

Comment:
Sorry, there must have been a miscommunication.  This entry didn't need to change, the link in the `bootstrapping.md` file needed to change to `/getting-started/setup/data-node-authentication`

Reply:
```suggestion
                        href: "data-node-authentication",
```